contracts/ClaimsManager.sol

```diff
@@ -508,36 +508,31 @@ contract ClaimsManager is
         }
     }
 
-    function timeOutClaim(uint256 claimIndex) external override {
+    function claimIsTimedOut(uint256 claimIndex)
```

Comment:
I considered using this in the logic above (like [here](https://github.com/api3dao/claims-manager/blob/004db8ecf0c009957bdbeb9997cc0f8e8cfb6196/contracts/ClaimsManager.sol#L311)) but figured it will be too gas-expensive